terra-jupyter-hail/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.0.16 - 06/27/2020
+- Upgrade `hail` to `0.2.50`
+
+Image URL `us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:0.0.16
```

Comment:
Missing a matching `
Also would be nice if you can link the hail change log here as well

Reply:
Done!

Reply:
thanks @johnc1231 !